github-dark.css

```diff
@@ -3585,6 +3585,17 @@
   .zh-collapse-control {
     color: #ccc !important;
   }
+
+  /* Issue/PR status text */
+  .zh-discussion-item .zh-pipeline-is,
+  .zh-discussion-item .zh-pipeline-was,
+  .zh-discussion-item .zh-estimate-is,
+  .zh-discussion-item .zh-estimate-was,
+  .zh-discussion-item .zh-issue,
+  .zh-discussion-item .zh-release{
+    color: #ccc !important;
+  }
+
```

Comment:
Hi @Geolim4!

Thanks for contributing. Please combine these selectors with the block above with the same color definition. Wrap the selectors at around 80 columns as well. Thanks!

Reply:
And remove the empty lines.

Reply:
Okay, I did not dare to do it :)

Reply:
hehe, it's fine. Hopefully this was explained well enough in our [contributing guide](https://github.com/StylishThemes/GitHub-Dark/blob/master/.github/CONTRIBUTING.md)?

Reply:
@Mottie Done, commit added.